### PR TITLE
add responsiveness to desktop version of portfolio

### PR DESCRIPTION
**Pull request by kalbek**
I've added a responsive layout according to project requirements on the desktop version of the personal portfolio project.

### CODE REVIEW

**Review by amon-cofie — CHANGES_REQUESTED** [review]
Hi @kalbek , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that does not have a descriptive pull request summary. Please add one and submit for another review.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: styles.css [hunk 1]
```diff
@@ -526,3 +550,339 @@ form input {
...
+
+.shape-9-desktop {
+  display: none;
+}
+
+.desktop-textarea {
+  display: none;
+}
+
+@media (min-width: 768px) {
```
Comment (NduatiKagiri) [inline styles.css:570]: - [ ] You have done a commendable job implementing the desktop view 💯, but please note that your mobile design should remain as perfect as it was before. Your Works section cards are now breaking in the mobile view and the left border is not visible. Kindly fix this and make sure both the mobile and desktop versions are perfect.

  kaleb1

**Review by NduatiKagiri — CHANGES_REQUESTED** [review]
Hi @kalbek 👋 ,

# CHANGES REQUIRED ♻️  

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

almost-there

## Highlights ✔️ 
✅ No Linters errors
✅ GitHub Flow observed
✅ Impressive desktop view

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: styles.css (hunk 1 above)
Comment (kalbek) [reply styles.css:570]: Kindly, I cannot see what is breaking in the mobile view, can you please be a bit more specific?

**Review by shella12 — APPROVED** [review]
Hi @kalbek , 


## Status : APPROVED 🥳 🥳 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights

- Linters are passing ✔️ 
- README is professional 👍 
- Implemented all requirements 👏 
- Clean code ✨ 
- Awesome implementation of UI design 🥇  

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**